source/mainnet/net/concepts/concepts-baker.rst

```diff
@@ -76,19 +78,49 @@ Finalization
+
+Finalizers sign a block, and their collective signatures are aggregated to form a :ref:`quorum certificate<glossary-quorum-certificate>`. This quorum certificate is then included in the next block. When two blocks that are parent-child are in consecutive rounds in the same epoch and both have a quorum certificate, then the block in the first of these rounds (together with its ancestors) is considered finalized. Why isn't the child block considered to be final if it has a QC? This is to cover edge cases where network delays cause the QC of a block to not be received by the next block producer before a timeout. In that case, the block gets skipped by the next block producer and it cannot be considered final. To resolve this, only  the first among two consecutive certified blocks is considered to be final.
+
+Finalization happens at a minimum one second after block creation. A new block has to be created descended from that block for finalization to happen.
+
 When a sufficiently large number of members of the committee have received the block and agree on its outcome, the block is finalized. Newer blocks must have the finalized block as an ancestor to ensure the integrity of the chain.
 
 Finalization committee
 ----------------------
 
-The finalization committee is formed by the bakers with a staked amount of at least 0.1% of the total amount of existing CCD. This specifically implies that in order to participate in the finalization committee you will probably have to modify the staked amount to reach the threshold.
+The finalization committee is formed by the bakers with a staked amount of at least 0.1% of the total stake of CCD in pools. This specifically implies that in order to participate in the finalization committee you will probably have to modify the staked amount to reach the threshold. There is a minimum and maximum size for the finalization committee; it is 0.1% of the stake in pools, but always at least x bakers and always at most y bakers (e.g., x = 25 and y = 500).
 
 Participating in the finalization committee produces rewards on each block that is finalized. The rewards are paid to the baker account some time after the block is finalized.
 
 Overview of the baker process
 =============================
 
+#. A Proof of stake based lottery system produces a list of bakers. The higher the baker’s stake, the higher the probability of being included on the list more often.
+
+#. The first baker on the list (Bob) makes a new block that appends to the genesis block.
+
+#. Bob then broadcasts the block to all peers.
+
+#. If the block is valid, the finalizers will sign it.
+
+#. If the combined effective stake of the finalizers who sign the block is *greater than or equal to* OR  *effective stake is at least* two-thirds of the total stake, the block gets a :ref:`Quorum Certificate (QC)<glossary-quorum-certificate>` that certifies that this is a valid block. Without the QC the new round cannot progress.
```

Comment:
I don't understand this OR statement. Aren't the two the same condition?

Reply:
It was based on an earlier review comment from @EmilLa1. Perhaps he can explain. I agree that they sound the same.

Reply:
Yes, my suggestion was just to use **one** of the two phrasings. :-) At least or greater than or equal to... Not both :)

Reply:
Ah, I didn't understand that from your suggestion. I will fix it.

Reply:
My bad!